.editorconfig

```diff
@@ -168,4 +168,7 @@ dotnet_naming_rule.members_should_be_pascal_case.style = pascal_case_style
 
 dotnet_naming_symbols.all_members.applicable_kinds = *
 
-dotnet_naming_style.pascal_case_style.capitalization = pascal_case
\ No newline at end of file
+dotnet_naming_style.pascal_case_style.capitalization = pascal_case
+
+# VSTHRD100: Avoid async void methods
+dotnet_diagnostic.VSTHRD100.severity = warning
```

Comment:
Why not pump it up all the way to an error? Is there ever a case where async void is acceptable? And where you wouldn't want to force the developer to explicitly justify it in situ with a `[SuppressMessage]` or `#pragma disable`?

Reply:
In my experience, people don't like that because it just gets in the way of development. The "standard" seems to be have things as warnings, and have CI enable `TreatWarningsAsErrors` so that CI fails, thus protecting main, but devs can still run tests/F5 without being annoyed.

Reply:
I won't push on it too much, but in my limited experience:
* there has never been a case where async void couldn't be done with void or async Task instead
* promoting to an error ensures it will never slip in later.